screen-capture-api/basic-screen-capture/index.css

```diff
@@ -0,0 +1,45 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0 25px;
+}
+
+#main-app {
+  display: flex;
+  gap: 5%;
+  min-width: 980px;
+}
+
+video,
+#demo {
+  flex: 1;
+}
+
+video,
+#demo>p,
+#log {
+  border: 1px solid #CCC;
+  margin: 0;
+}
+
+video {
+  max-width: 50%;
+  aspect-ratio: 4/3;
+}
+
+#demo>h2 {
+  margin-top: 0;
+}
+
+#demo>p {
+  padding: 5px;
+  height: 320px;
+}
+
+#log {
+  height: 20rem;
```

Comment:
Maybe because of this, but the logs go a bit beyond the bottom of the viewport here:

![image](https://github.com/user-attachments/assets/cf50e6f9-6985-436a-a2ed-a033961cfb99)

Reply:
I think that's inevitable really.

Reply:
Not a blocker, just FYI